Refit/RequestBuilderImplementation.cs

```diff
@@ -170,8 +170,16 @@ void AddMultipartItem(MultipartFormDataContent multiPartContent, string fileName
                             return (T)(object)await reader.ReadToEndAsync().ConfigureAwait(false);
                         }
 
-                        using(var jsonReader = new JsonTextReader(reader))
+                        using (var jsonReader = new JsonTextReader(reader))
                         {
+                            if (restMethod.SerializedReturnType.IsConstructedGenericType &&
+                                restMethod.SerializedReturnType.GetGenericTypeDefinition() == typeof(RefitResponse<>).GetGenericTypeDefinition())
```

Comment:
Should be IsAssignableTo `IRefitResponse<T>` or it'll miss other implementation types.

Reply:
I'm not sure there's a point in that interface at all -- there's only ever one implementation, right? And it's just a DTO.

Reply:
@bennor  Removing the interface as suggested.
@onovotny restMethod.SerializedReturnType.IsAssignableFrom(typeof(ApiResponse<>)) returns false due T is ApiResponse<MyClass>.

Reply:
It's moot if it's a struct with one impl now.